src/pymovements/datasets/raccoons.py

```diff
@@ -0,0 +1,225 @@
+        },
+    )
+    mirrors: dict[str, list[str]] = field(
+        default_factory=lambda: {
+            'gaze': [
+                'https://data.ru.nl/api/collectionfiles/ru/id/ru_395469/files/',
+            ],
+            'precomputed_events': [
+                'https://data.ru.nl/api/collectionfiles/ru/id/ru_395469/files/',
+            ],
+            'precomputed_reading_measures': [
+                'https://data.ru.nl/api/collectionfiles/ru/id/ru_395469/files/',
+            ],
+        },
+    )
+    resources: dict[str, list[dict[str, str | None]]] = field(
+        default_factory=lambda: {
+            'gaze': [
+                {
+                    'resource': 'download/eyetracking%2FET_raw_data.zip',
+                    'filename': 'ET_raw_data.zip',
+                    'md5': None,  # type: ignore
+                },
+            ],
+            'precomputed_events': [
+                {
+                    'resource': 'download/eyetracking%2FET_fix_data.tsv',
+                    'filename': 'ET_fix_data.tsv',
+                    'md5': None,  # type: ignore
+                },
+            ],
+            'precomputed_reading_measures': [
+                {
+                    'resource': 'download/eyetracking%2FET_word_data.tsv',
+                    'filename': 'ET_word_data.tsv',
+                    'md5': None,  # type: ignore
+                },
+            ],
+        },
+    )
+
+    extract: dict[str, bool] = field(
+        default_factory=lambda: {
+            'gaze': True,
+            'precomputed_events': False,
+            'precomputed_reading_measures': False,
+        },
+    )
+
+    experiment: Experiment = field(
+        default_factory=lambda: Experiment(
+            screen=Screen(
+                width_px=1920,
+                # in the paper its written 1018, but assume typo
+                height_px=1080,
+                width_cm=56.8,
+                height_cm=33.5,
+                distance_cm=105.5,
+                origin='center',
+            ),
+            eyetracker=EyeTracker(
+                sampling_rate=1000,
+                # double check once data is back up
+                left=True,
+                right=False,
+                model='EyeLink 1000 Plus',
+                vendor='EyeLink',
+                mount='Desktop',
+            ),
+        ),
+    )
+
+    filename_format: dict[str, str] = field(
+        default_factory=lambda: {
+            'gaze': r'{participant_id:s}.asc',
+            'precomputed_events': r'ET_fix_data.tsv',
+            'precomputed_reading_measures': r'ET_word_data.tsv',
+        },
+    )
+
+    filename_format_schema_overrides: dict[str, dict[str, type]] = field(
+        default_factory=lambda: {
+            'gaze': {'participant_id': str},
+            'precomputed_events': {},
+            'precomputed_reading_measures': {},
+        },
+    )
+
+    trial_columns: list[str] = field(default_factory=lambda: [])
```

Comment:
@SiQube I just noticed that I forgot to add `trial_index0` to the trial columns here. Although I'm not sure we should add it actually, because it only exists in the gaze dataframe, but not in the event dataframe?

Reply:
Maybe `trial_columns="trial_index"` should only be added to the `custom_read_kwargs` for gaze?